ovn-tester/cms/openstack/openstack.py

```diff
@@ -234,6 +241,38 @@ def add_internal_network_to_project(
         if snat_port is not None:
             self.nbctl.nat_add(project.router, snat_port.ip, snated_network)
 
+    def _get_gateway_chassis(self, count: int = 1) -> List[ChassisNode]:
+        """Return list of Gateway Chassis with size defined by 'count'.
+
+        Chassis are picked at random from all available 'worker_nodes' to
```

Comment:
Nit: I understand the intent but on the other hand, this is a testing tool and reproducible runs are very important .  Not necessarily in this PR but do we need to log the chosen values and provide a way to explicitly select the same chassis if we want to re-run the exact same workload while debugging a DUT issue?

Reply:
This is a very good point. I originally intended to do even distribution of GWs but the implementation turned out more complicated than I anticipated so I went with random. I'll open a ticket about this so that the issue is not lost.

Reply:
Thanks @mkalcok